tm2py/components/network/create_tod_scenarios.py

```diff
@@ -237,18 +238,21 @@ def _create_transit_scenarios(self):
             # set fixed guideway times, and initial free flow auto link times
             # TODO: cntype_speed_map to config
             cntype_speed_map = {"CRAIL": 45.0, "HRAIL": 40.0, "LRAIL": 30.0, "FERRY": 15.0}
+            walk_speed = 3.0
+            transit_speed = 30.0

```

Comment:
Can we expose these in model_config.toml?

Reply:
added to model_config.toml in https://github.com/camsys/tm2py/pull/53/commits/cfce21695d62ed72e73dd31a858b23d9291cda59